apps/etterlatte-utbetaling/src/main/kotlin/no/nav/etterlatte/utbetaling/config/ApplicationProperties.kt

```diff
@@ -62,10 +62,10 @@ data class ApplicationProperties(
                     serviceUserUsername = requireEnvValue(srvuser),
                     serviceUserPassword = requireEnvValue(srvpwd),
                     leaderElectorPath = requireEnvValue(ELECTOR_PATH),
-                    grensesnittavstemmingEnabled = getValue(GRENSESNITTAVSTEMMING_ENABLED).toBoolean(),
```

Comment:
Er dette noe vi vil ha som required @henrikg ?

Reply:
Han er iallfall definert i både dev.yaml, prod.yaml og i .run-fila for lokal køyring, så i praksis er han tilgjengeleg i dag. Korvidt vi eigentleg treng denne brytaren lenger er eg meir usikker på.. 🤔

Reply:
@sebassonav : Den er vel i praksis required ja, så kan godt settes som det. Når det gjelder om det er behov for bryteren lenger, er det en annen sak - tenker det kan være greit å ha fleksibiliteten.